runme.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+oidc-agent > /tmp/oidc-agent.env
+
+sleep infinity
+
+exec "$@"
```

Comment:
```suggestion
```

Because you sleep the process forever right before this, the `exec` line is unnecessary